deploy/helm/groundlight-edge-endpoint/files/nginx.conf

```diff
@@ -0,0 +1,37 @@
+events {
```

Comment:
Seems like while we're in this transition phase between setup-ee and helm that we're keeping a lot of duplicate/near-duplicate files. Just making a note to myself as something to be aware of when reviewing code until setup-ee is removed.

Reply:
Yes, exactly. That's not really an ideal situation, but I don't want to break the "old way" of doing it until I have the GLHub stuff doing it the "new way" and we've proven out/debugged all the other use cases (like the Jetson and UR stuff).